_config.yml

```diff
@@ -117,4 +117,4 @@ compress_html:
         - development
         - stage
 
-exclude: ["travis.sh", "Rakefile", "README.md", "Gemfile", "Gemfile.lock", "FAQ.md"]
+exclude: ["travis.sh", "Rakefile", "README.md", "Gemfile", "Gemfile.lock", "FAQ.md", "vendor"]
```

Comment:
Had to add this to fix this annoying issue. https://github.com/jekyll/jekyll/issues/2938